cartographer_ros/CMakeLists.txt

```diff
@@ -47,6 +47,8 @@ find_package(tf2_ros REQUIRED)
 find_package(tf2_eigen REQUIRED)
 find_package(tf2_msgs REQUIRED)
 find_package(visualization_msgs REQUIRED)
+find_package(rosbag2_cpp)
+find_package(urdf)
```

Comment:
Both of these dependencies need to be declared in the package.xml for this package, otherwise the CI server will never install them.

Reply:
Thanks very much for your help